src/series/boxPlot.js

```diff
@@ -0,0 +1,86 @@
+import d3 from 'd3';
+import dataJoinUtil from '../util/dataJoin';
+import {noop} from '../util/fn';
+import svgBoxPlot from '../svg/boxPlot';
+import {rebindAll} from '../util/rebind';
+import boxBase from './boxBase';
+
+export default function() {
+
+    var decorate = noop,
+        barWidth = 5,
+        orient = 'vertical',
+        base = boxBase();
+
+    var dataJoin = dataJoinUtil()
```

Comment:
@jdunkerley - just wondering if you made any more progress on your datajoin utilities? I'd love to see this repeated pattern eliminated!